notebooks/HASP/FluxScaleTutorial/FluxScaleTutorial.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Please move the imports to the top of the notebook and remove the duplicated import os.</p><p>The&nbsp;<code>import glob as glob</code>&nbsp;is redundant and can be simplified to&nbsp;<code>import glob</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>